main.go

```diff
@@ -1,8 +1,15 @@
 // Package main provides the entrypoint for the sops-compliance-checker executable.
 package main
 
-import "fmt"
+import (
+	"fmt"
+	"log/slog"
+	"os"
+)
 
 func main() {
+	logger := slog.New(slog.NewJSONHandler(os.Stdout, nil))
```

Comment:
I know it's not really important right now since we don't have anything runnable, but I'd probably go with `slog.NewTextHandler` if we decide to stick with `slog`. Mainly because the logs will not be consumed by a log collection system, but are intended to be read by the user. And there JSON makes them a bit harder to read.

Feel free to ignore for now.